core/ops_builtin.rs

```diff
@@ -43,9 +44,33 @@ deno_core::extension!(
     op_write_type_error,
     op_shutdown,
     op_format_file_name,
-    op_is_proxy,
     op_str_byte_length,
     op_panic,
+    ops_builtin_types::op_is_any_array_buffer,
```

Comment:
We should add `op_is_array_buffer_view`, `op_is_data_view` and `op_is_typed_array` -- those three ops could be very useful for fast-op dispatch selection.